src/playlist-loader.js

```diff
@@ -558,14 +587,14 @@ export default class PlaylistLoader extends EventTarget {
         uri: window.location.href,
         playlists: [{
           uri: this.srcUrl,
-          id: 0
+          id: 0,
+          resolvedUri: this.srcUrl,
```

Comment:
should the faked `resolvedUri` for the master playlist also be run through `resolveManifestRedirect`?

Reply:
It should have already been set correctly above on line 558.